pkg/health/health_argo.go

```diff
@@ -0,0 +1,44 @@
+package health
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+type nodePhase string
```

Comment:
I've made the types and constants private since they don't need to be exported from this library